### ISSUE TITLE
Double enabling_status not working

### ISSUE THREAD

**Issue by pietrobalatti** [ADVRHumanoids/XBotGUI#41]
In case the same enabling status is in more than one command_block the GUI is showing as enabled just the first one. More specifically:
```xml
<command_block>
    <command type="cmd_service" name="LSoftHand" enabling_status="RUNNING:HANDSELECTION"/>
    <command type="cmd_service" name="RSoftHand" enabling_status="RUNNING:HANDSELECTION"/>
</command_block>
```
when HANDSELECTION status is running just LSoftHand is enabled.

**Comment by pietrobalatti** [comment]
I noticed another weird behavior, when the FSM goes from GRASP to HOMING_LEE the "Handover_success" button becomes active but the "After_handover" remains active (it should be active just in the GRASP state).

**Comment by alessandrosettimi** [comment]
I think I found the bug, should be fixed in a59f81ce5055c44c08e47134af18b0c896c7561e.

Can you test a bit?

**Comment by pietrobalatti** [comment]
Solved, thanks!
